sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -132,6 +132,27 @@ case class AlterDatabasePropertiesCommand(
   }
 }
 
+/**
+ * A command for users to set new location path for a database
+ * If the database does not exist, an error message will be issued to indicate the database
+ * does not exist.
+ * The syntax of using this command in SQL is:
+ * {{{
+ *    ALTER (DATABASE|SCHEMA) database_name SET LOCATION path
+ * }}}
+ */
+case class AlterDatabaseLocationCommand(databaseName: String, location: String)
```

Comment:
Let us follow the name AlterTableSetLocationCommand. Rename it to AlterDatabaseSetLocationCommand